app/src/main/res/layout/fragment_request.xml

```diff
@@ -1,73 +1,78 @@
+	<androidx.constraintlayout.widget.ConstraintLayout
+		android:layout_width="match_parent"
+		android:layout_height="match_parent"
+		android:padding="@dimen/spacing_large">
 
-	<Button
-		android:id="@+id/vSearchButton"
-		android:layout_width="0dp"
-		android:layout_height="wrap_content"
-		android:text="@string/fragment_request_search"
-		app:layout_constraintBottom_toTopOf="@id/vPageTitle"
-		app:layout_constraintEnd_toEndOf="parent"
-		app:layout_constraintStart_toStartOf="parent"
-		app:layout_constraintTop_toBottomOf="@id/vIdInput" />
+		<EditText
+			android:id="@+id/idInput"
+			android:layout_width="0dp"
+			android:layout_height="wrap_content"
+			android:hint="@string/fragment_request_post_id"
+			android:importantForAutofill="no"
+			android:inputType="number"
+			app:layout_constraintBottom_toTopOf="@id/searchButton"
+			app:layout_constraintEnd_toEndOf="parent"
+			app:layout_constraintStart_toStartOf="parent"
+			app:layout_constraintTop_toTopOf="parent"
+			app:layout_constraintVertical_bias="0"
+			app:layout_constraintVertical_chainStyle="packed" />
 
-	<TextView
-		android:id="@+id/vPageTitle"
-		android:layout_width="wrap_content"
-		android:layout_height="wrap_content"
-		android:layout_margin="@dimen/spacing_medium_more"
-		android:textAlignment="center"
-		android:textSize="@dimen/font_big"
-		app:layout_constraintBottom_toTopOf="@id/vPageBody"
-		app:layout_constraintEnd_toEndOf="parent"
-		app:layout_constraintStart_toStartOf="parent"
-		app:layout_constraintTop_toBottomOf="@id/vSearchButton"
-		tools:text="Titulo de la página" />
+		<Button
+			android:id="@+id/searchButton"
+			android:layout_width="0dp"
+			android:layout_height="wrap_content"
+			android:text="@string/fragment_request_search"
+			app:layout_constraintBottom_toTopOf="@id/pageTitle"
+			app:layout_constraintEnd_toEndOf="parent"
+			app:layout_constraintStart_toStartOf="parent"
+			app:layout_constraintTop_toBottomOf="@id/idInput" />
 
-	<TextView
-		android:id="@+id/vPageBody"
-		android:layout_width="wrap_content"
-		android:layout_height="wrap_content"
-		android:layout_margin="@dimen/spacing_medium_more"
-		android:textAlignment="center"
-		android:textSize="@dimen/font_medium_extra"
-		app:layout_constraintBottom_toBottomOf="parent"
-		app:layout_constraintEnd_toEndOf="parent"
-		app:layout_constraintStart_toStartOf="parent"
-		app:layout_constraintTop_toBottomOf="@id/vPageTitle"
-		tools:text="Descripción de la página" />
+		<TextView
+			android:id="@+id/pageTitle"
+			android:layout_width="wrap_content"
+			android:layout_height="wrap_content"
+			android:layout_margin="@dimen/spacing_medium_more"
+			android:textAlignment="center"
+			android:textSize="@dimen/font_big"
+			app:layout_constraintBottom_toTopOf="@id/pageBody"
+			app:layout_constraintEnd_toEndOf="parent"
+			app:layout_constraintStart_toStartOf="parent"
+			app:layout_constraintTop_toBottomOf="@id/searchButton"
+			tools:text="Titulo de la página" />
```

Comment:
check the tabulation here. You should execute a `clean code` command

Reply:
The tabulation here is ok. It was wrong before.